pkg/unikontainers/hypervisors/firecracker.go

```diff
@@ -88,9 +89,18 @@ func (fc *Firecracker) Execve(args ExecArgs) error {
 	}
 
 	// VM config for Firecracker
+	mem := uint(256)
+	if args.MemSizeMiB != "" {
+		memInt, err := strconv.Atoi(args.MemSizeMiB)
+		if err != nil {
+			vmmLog.WithError(err).Error("Failed to parse memory size")
```

Comment:
Maybe it would be a good idea to improve the error message notifying that we will use the default value.

Reply:
You are right, I changed it.